src/Components/Components/src/Reflection/MemberAssignment.cs

```diff
@@ -16,13 +15,13 @@ internal class MemberAssignment
             [DynamicallyAccessedMembers(Component)] Type type,
             BindingFlags bindingFlags)
         {
-            var dictionary = new Dictionary<string, List<PropertyInfo>>();
+            var dictionary = new Dictionary<string, List<PropertyInfo>>(StringComparer.Ordinal);
```

Comment:
Why pass in the comparer explicitly?  It's not a problem, I'm just wondering what it's fixing.  Just for clarity?